content/en/docs/demo/services/_index.md

```diff
@@ -15,10 +15,11 @@ To visualize request flows, see the [Service Diagram](../architecture/).
 | [currencyservice](currency/)              | C++           | Converts one money amount to another currency. Uses real values fetched from European Central Bank. It's the highest QPS service.    |
 | [emailservice](email/)                    | Ruby          | Sends users an order confirmation email (mock/).                                                                                     |
 | [frauddetectionservice](fraud-detection/) | Kotlin        | Analyzes incoming orders and detects fraud attempts (mock/).                                                                         |
-| [frontend](frontend/)                     | JavaScript    | Exposes an HTTP server to serve the website. Does not require sign up / login and generates session IDs for all users automatically. |
+| [frontend](frontend/)                     | TypeScript    | Exposes an HTTP server to serve the website. Does not require sign up / login and generates session IDs for all users automatically. |
```

Comment:
unrelated but I think this is considered TypeScript now?

Reply:
yes! Thank you